.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ orbs:
 executors:
   xcode_15:
     macos:
-      xcode: 15.1.0
+      xcode: 16.1.0
```

Comment:
oh nice, i missed that the previous update didn't already do this

Reply:
Didn't think about it until `nonisolated(unsafe)` caused the build to fail 😅